### ISSUE TITLE
I see no room descriptions in either the Main window or the Room window.

### ISSUE THREAD

**Issue by tbgg** [joemcbride/outlander-osx#114]
I'm on MacOS Sierra and the latest version of Outlander.  (0.10.8, I think)

For the most part I can use the automapper to get around, but not being able to see room descriptions will be a problem for quests.

Also, if I or another person says anything, neither the word 'You' or the name of the person speaking or the word 'says' appears in the Conversation or Talk window (I can't tell which it is) or the main window.

I see stuff like this:
        , "Test."  (Text is 'You say, "Test."')
            , "Eh?"  (Actual text is 'Malciss asks, "Eh?"')

For both the room description text and the speech text, if I highlight the text, I can see it, and I can copy/paste it fine.  So it seems to be there, it just shows as black on black until it's highlighted.

I don't know if this matters, but I do have color highlights that would apply in these two windows, but at least in the Room window they appear fine.  (Some of the highlights I made don't appear in the Main window even though the text does, but I figure that's a separate bug.)

The specific color highlights I use are 
^Obvious (exits|paths):.*\.$ (#00FF00)
^Also here.*\.$ (#FFFF66)
^You also see.*\.$  (#4CB74C)
^Mapped exits:.*\.$  (#8EEEE)
These are 80% of the full line highlights I have set up thus far.

If I remove the ^ and the .*\.$ and just highlight the indicated text, the highlights appear correctly in both windows, but the room descriptions and identifying speech text remain black on black.

**Comment by joemcbride** [comment]
Can you post the contents of your `presets.cfg` file?  It should be found in `~/Documents/Outlander/Config/Profiles/Default/presets.cfg`.

The defaults should look something like this:

```
#preset {automapper} {#FFFF00}
#preset {chatter} {#99FFFF}
#preset {creatures} {#FFFF00}
#preset {roomdesc} {#cccccc}
#preset {roomname} {#0000FF}
#preset {scriptecho} {#00ffff}
#preset {scripterror} {#efefef,#ff3300}
#preset {scriptinfo} {#0066cc}
#preset {scriptinput} {#acff2f}
#preset {sendinput} {#acff2f}
#preset {speech} {#99FFFF}
#preset {thought} {#99FFFF}
#preset {whisper} {#99FFFF}
```

I would look and see if perhaps your `speech` and/or `whisper` presets are a dark color.

**Comment by tbgg** [comment]
I copied in a bunch of .cfg files from Genie, and it was very different from what you show, though some things were named the same.  In some cases, it even referred to color names like Navy, MediumBlue, Purple, etc. instead of the hex codes.  I changed mine to match what yours shows, and will probably go back and make sure I have all of the windows covered.  Thanks so much, as I'm starting to feel like I might be able to play now!   ;)
